### DISCUSSION TITLE
When new release will be created ?

### DISCUSSION THREAD

**Discussion by vivanov83** [rancher-sandbox/rancher-desktop#602] [General]
How often a release will be created, weekly, monthly or when changes are merged ?

**Comment by jandubois** [comment]
There is no fixed cadence, but I would expect roughly monthly releases for a while. But sometimes it takes a little longer to stabilize bigger changes, and sometimes there are bugs affecting many users, requiring an earlier release.

Sorry for giving kind of a non-answer, but we simply often don't know in advance.
